### DISCUSSION TITLE
Lora fp16 1.28s/it ... i think is a slow.

### DISCUSSION THREAD

**Discussion by salocinmad** [bmaltais/kohya_ss#2409] [Q&A]
Hi world. I need help

I have a windows 11 with 64 GB RAM, Nvidia RTX 3060 16GB.
Nvidia CUDA 12.4 Update 1  -> Installed

I get 1,3s/i (seconds interactions). I think this is a slow... becouse i view youtube videos go more fast with similar hardware specs...
Like i read, lora is more fast than dreambooth, but i get more speed in dreambooth with same parameters... I think i have a problem with something but i dont find it.

I reinstall cuda, previously I remove all system variables about CUDA

**Lora [1.27-1.3] s/it (seconds 1 interaction)**
Dreamsbooth [2.5-2.75] it/s  (interactions seconds)
caching latents [4.2-4.4] it/s  (interactions seconds)

Lora train screenshot
<img width="1272" alt="2024-04-28 215240" src="https://github.com/bmaltais/kohya_ss/assets/59669036/1e70fb5c-b2fe-4806-a7d3-08e958961e02">

CuDDN files -> Installed via setup.py
bitsandbytes -> installed windows version

I try to train a lora 
- AdamW8bit
- LR scheduler constant
- fp16
- sd 1.5
- 18 pics
- 40 repeats
- max resolution 512,512
- Full command in attachment (further down in this post)

[command.txt](https://github.com/bmaltais/kohya_ss/files/15143497/command.txt)

**Comment by dmaltsevQA** [comment]
Disty0 wrote earlier:
Checks out with these benchmark:
https://www.pugetsystems.com/labs/articles/stable-diffusion-lora-training-consumer-gpu-analysis/

**Comment by salocinmad** [comment]
I check it.. and i think is a very bad ...

Now im testing in a runpod with a 3090 TI, and i think i have some bad in my config...

Some body can check my command.txt for check the values i use?? maybe i dont understan something and i put a incorrect parameters..
